api/libproject/devstart.rb

```diff
@@ -46,69 +47,60 @@ def parse args
 def dev_up(*args)
   common = Common.new
 
-  account = get_auth_login_account()
-  if account == nil
-    raise("Please run 'gcloud auth login' before starting the server.")
-  end
+  ensure_gcloud_account
+  Credentials.new.maybe_download_sa :dev
 
   at_exit { common.run_inline %W{docker-compose down} }
-  common.status "Starting database..."
-  common.run_inline %W{docker-compose up -d db}
+  start_db
   common.status "Running database migrations..."
   common.run_inline %W{docker-compose run db-migration}
 
   common.status "Updating configuration..."
   common.run_inline %W{docker-compose run update-config}
 
-  run_api(account)
+  run_api
 end
 
-def run_api(account)
+def start_db()
   common = Common.new
-  # TODO(dmohs): This can be simplified now that we are using a shared service account.
-  do_run_with_creds("all-of-us-workbench-test", account, nil, lambda { |project, account, creds_file|
-    common.status "Starting API. This can take a while. Thoughts on reducing development cycle time"
-    common.status "are here:"
-    common.status "  https://github.com/all-of-us/workbench/blob/master/api/doc/2017/dev-cycle.md"
-    at_exit { common.run_inline %W{docker-compose down} }
-    common.run_inline_swallowing_interrupt %W{docker-compose up api}
-  })
+  common.status "Starting database..."
+  common.run_inline %W{docker-compose up -d db}
 end
 
-def clean()
-  common = Common.new
-  common.run_inline %W{docker-compose run --rm api ./gradlew clean}
+def run_api_and_db(*args)
+  start_db
+  run_api
 end
 
-def run_api_and_db(*args)
+def run_api()
   common = Common.new
-  account = get_auth_login_account()
-  if account == nil
-    raise("Please run 'gcloud auth login' before starting the server.")
-  end
-  common.status "Starting database..."
-  common.run_inline %W{docker-compose up -d db}
-  run_api(account)
+  common.status "Starting API. This can take a while. Thoughts on reducing development cycle time"
```

Comment:
This used to run inside do_run_with_creds, which ensures we activate the service account gsutil uses to copy the creds files from GCS:

https://github.com/all-of-us/workbench/blob/master/api/libproject/devstart.rb#L265

That doesn't seem to be happening anymore... which makes me think this will only work if you have already activated the service account in question locally. (Or is gsutil using other creds somehow?)

Please use "gcloud auth revoke --all" before running stuff to make sure it still works...